common/JailUtil.cpp

```diff
@@ -581,13 +581,13 @@ bool updateDynamicFilesImpl(const std::string& sysTemplate)
 
         if (checkWritableSysTemplate && !FileUtil::isWritable(sysTemplate))
         {
-            disableBindMounting(); // We can't mount from incomplete systemplate that can't be updated.
             LinkDynamicFiles = false;
             LOG_WRN("The systemplate directory ["
                     << sysTemplate << "] is read-only, and at least [" << dstFilename
-                    << "] is out-of-date. Will have to copy sysTemplate to jails. To restore "
-                       "optimal performance, make sure the files in ["
-                    << sysTemplate << "/etc] are up-to-date.");
+                    << "] is out-of-date. Will have to clone dynamic elements of "
+                    << "systemplate to the jails.. To restore optimal performance, "
```

Comment:
`..` duplicated.

Is there really a performance penalty?

Reply:
Well, there is a perf penalty if we end up doing the mega copy of the entire /opt/cool/systemplate in the end. If we only need to make a /etc, then my one is about a 1M so not a big deal I suppose, but it would be better to not do any copy. I just wanted to update this message to reflect what will really happen after this change.